parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -246,14 +245,15 @@ public <D> FileAppender<D> build() throws IOException {
       int rowGroupSize = context.rowGroupSize();
       int pageSize = context.pageSize();
       int pageRowLimit = context.pageRowLimit();
-      boolean dictionaryEnabled = context.dictionaryEnabled();
       int dictionaryPageSize = context.dictionaryPageSize();
       String compressionLevel = context.compressionLevel();
       CompressionCodecName codec = context.codec();
       int rowGroupCheckMinRecordCount = context.rowGroupCheckMinRecordCount();
       int rowGroupCheckMaxRecordCount = context.rowGroupCheckMaxRecordCount();
       int bloomFilterMaxBytes = context.bloomFilterMaxBytes();
       Map<String, String> columnBloomFilterEnabled = context.columnBloomFilterEnabled();
+      boolean enableDictionary =
+          Boolean.valueOf(config.getOrDefault(ParquetOutputFormat.ENABLE_DICTIONARY, "true"));
```

Comment:
[minor] if we going to a new line we can use : 
```java
PropertyUtil.propertyAsBoolean(config, ParquetOutputFormat.ENABLE_DICTIONARY, true);
```

Reply:
Updated, unfortunately still goes to a newline but it's cleaner!